test/models/player_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class PlayerTest < ActiveSupport::TestCase
```

Comment:
I does nothing right now I suppose. I believe tests for the player model should go here but I don't feel confident it writing them. What do you advise?